doc/sphinx/source/recipes/recipe_flato13ipcc.rst

```diff
@@ -434,6 +544,18 @@ References
   variations to anthropogenic and natural causes using CMIP5 simulations. Journal
   of Geophysical Research: Atmosphere, 118, 4001-4024, doi:10.1002/jgrd.50239.
 
+* Seneviratne, S. I., Nicholls, N., Easterling, D., Goodess, C. M., Kanae, S., 
+  Kossin, J., Luo, Y., Marengo, J., McInnes, K., Rahimi, M., Reichstein, M., 
+  Sorteberg, A., Vera, C., and Zhang, X., 2012:
+  Changes in climate extremes and their impacts on the naturalphysical 
+  environment. , in: Managing the Risks of Extreme Events and Disasters to 
+  Advance Climate Change Adaptation.  A Special Report of Working Groups I and 
+  II of the Intergovernmental Panel on ClimateChange (IPCC), edited by: 
+  Field, C. B., Barros, V., Stocker, T. F., Qin, D., Dokken, D. J., 
+  Ebi, K. L., Mastrandrea, M. D., Mach, K. J., Plattner, G.-K., Allen, S. K., 
+  Tignor, M., and Midgley, P. M., Cambridge University Press, Cambridge, UK, 
+  and New York, NY, USA, 109-230.
+
```

Comment:
I'm not sure: This reference is so far not in the recipe, because it is not a reference to any of the figures reproduced but only to define the regions in one Figure (this reference is cited within Flato et al. 2013). I think it is important to write it in the documentation, because the recipe produces figures for similar but not identical regions (the same way as it is done in Flato et al. 2013). 
I added this reference now as bibtex (seneviratne12ipcc.bibtex) and in the diagnostic but not in the recipe_flato13ipcc.yml.